docs/decisions/0004-downstream-effect-events.rst

```diff
@@ -56,21 +57,40 @@ Downstream effects that are not being implemented as part of this decision:
 
     * We will implement this later since we do not believe this to be an essential feature.
 
-For easier visualization, here are all of the events we plan to implement, described from end to end:
+For easier visualization, here are all of the downstream effects we plan to port, described from end to end:
+
+Downstream effects to be implemented as part of this decision:
```

Comment:
NIT: after seeing this table rendered. The general context / expected result columns are way more useful than the exact files/functions we are calling but get cutoff when viewing the page. I'd move those left so people can read them easily. TBH the file/function might not even be worth it if it doesn't really fit on the page.